sdk/formrecognizer/Azure.AI.FormRecognizer/tests/ReceiptClient/ReceiptClientLiveTests.cs

```diff
@@ -0,0 +1,141 @@
+        private const string ApiKeyEnvironmentVariableName = "FORM_RECOGNIZER_API_KEY";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ReceiptClientLiveTests"/> class.
+        /// </summary>
+        /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
+        public ReceiptClientLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        /// <summary>
+        /// Verifies that the <see cref="ReceiptClient" /> is able to connect to the Form
+        /// Recognizer cognitive service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        [Ignore("The receipt file has not been uploaded yet.")]
+        public async Task StartExtractReceiptsPopulatesExtractedReceipt(bool useStream)
+        {
+            var client = CreateInstrumentedClient();
+            Operation<IReadOnlyList<ExtractedReceipt>> operation;
+
+            if (useStream)
+            {
+                using var stream = new FileStream(@"", FileMode.Open);
+                operation = await client.StartExtractReceiptsAsync(stream, ContentType.Jpeg);
+            }
+            else
+            {
+                var endpoint = new Uri("");
+                operation = await client.StartExtractReceiptsAsync(endpoint);
+            }
+
+            await operation.WaitForCompletionAsync();
+
+            Assert.IsTrue(operation.HasValue);
+
+            var receipt = operation.Value.Single();
+
+            // The expected values are based on the values returned by the service, and not the actual
+            // values present in the receipt. We are not testing the service here, but the SDK.
+
+            Assert.AreEqual(1, receipt.StartPageNumber);
+            Assert.AreEqual(1, receipt.EndPageNumber);
+
+            Assert.AreEqual("Contoso Contoso", receipt.MerchantName);
+            Assert.AreEqual("123 Main Street Redmond, WA 98052", receipt.MerchantAddress);
+            Assert.AreEqual("123-456-7890", receipt.MerchantPhoneNumber);
+
+            Assert.IsNotNull(receipt.TransactionDate);
+            Assert.IsNotNull(receipt.TransactionTime);
+
+            var date = receipt.TransactionDate.Value;
+            var time = receipt.TransactionTime.Value;
+
+            Assert.AreEqual(10, date.Day);
+            Assert.AreEqual(6, date.Month);
+            Assert.AreEqual(2019, date.Year);
+
+            Assert.AreEqual(13, time.Hour);
+            Assert.AreEqual(59, time.Minute);
+            Assert.AreEqual(0, time.Second);
+
+            var expectedItems = new List<(int? Quantity, string Name, float? Price, float? TotalPrice)>()
+            {
+                (null, "8GB RAM (Black)", null, 999.00f),
+                (1, "SurfacePen", null, 99.99f)
```

Comment:
Validating all these fields looks fine for right now.  The design will change in Preview 2 so that we'll have client code we want to test around each of the values.  We'll need to do another pass over the tests then, because the client structure will be changing as well.